index.d.ts

```diff
@@ -746,6 +745,25 @@ declare module 'fastest-validator' {
 		messages?: MessagesType,
 	};
 
+	interface CompilationRule {
```

Comment:
Please add to the current section comments for a better understanding of what it doing.

Reply:
Because right now it looks like that `CompilationRule` required for `CompilationFunction`, and `CompilationFunction` required for `CompilationRule`

Reply:
- `CompilationFunction` - is used for [custom validator creation.](https://github.com/icebob/fastest-validator#custom-validator). 
- `CompilationRule` - `{ messages, schema }` in this example. I've added fields according to source code. It also includes link to `CompilationFunction`

Reply:
I forgot to change `add` method signature. Already fixed.